doc/classes/VoxelToolLodTerrain.xml

```diff
@@ -16,6 +16,7 @@
 			<param index="1" name="transform" type="Transform3D" />
 			<param index="2" name="area_size" type="Vector3" />
 			<description>
+				Allows using a [VoxelGeneratorGraph] as a brush, which opens possibility for various advanced scenarios. See [related article](../Generators.md#using-voxelgeneratorgraph-as-a-brush).
```

Comment:
`[related` has no precedent in Godot's documentation. XML files follow features supported by Godot, because this is the same file used to display API docs in the editor. 
Maybe you should use the `[url=https://...]text[/url]` syntax?

Reply:
Oof, yeah. 'related' is not intended as a keyword here, it's just "related article" x) I'll try the new syntax and see if the bbcode converter still complains at me.^^